nautobot/extras/filters.py

```diff
@@ -418,55 +431,181 @@ class Meta:
 )
 
 
-class CustomFieldFilter(django_filters.Filter):
+class CustomFieldFilterMixin:
```

Comment:
Any concerns about removing the `CustomFieldFilter` name entirely? I don't *think* we should have plugins importing and using it, but thought I should ask.

Reply:
I only see `CustomFieldFilter` in 3 private ntc github repositories and they're all forks of netbox or nautobot. I think it's safe to remove.